backend/src/rules/alter_card.clj

```diff
@@ -5,7 +5,7 @@
   [game-state path new-values]
   (update-in game-state path #(merge % new-values)))
 
-(defn relative-power
+(defn increase-power
   "Alters a cards' power, by adding the passed value to it"
```

Comment:
I'd comment here that it can "increase" negative values (therefore decreasing it)